packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -197,6 +193,18 @@ export default {
     }
 }
 
+// when the off-screen navigation is active (on mobile), it fixes to the top of the screen.
+// this stops the content being forced upwards when this happens (preventing slight visual glitch)
+.c-header--navInView {
```

Comment:
Was there a reason for changing the naming of this? As it's a JS state, we tend to use the `is-blah` prefix for these rather than modifiers (which tend to be just on or off when rendered on the page)

Reply:
I've used a global selector that fixes this now and it all looks good, thanks 👍